packages/ui/src/components/filters/CheckboxFilter.tsx

```diff
@@ -78,40 +78,37 @@ const CheckboxFilter: React.FC<TermFilterProps> = ({
                             onClick={() => onChange(filterGroup, [])}
                             type="text"
                         >
-                            {get(dictionary, 'actions.none', 'none')}
+                            {get(dictionary, 'actions.none', 'None')}
                         </Button>
                     </StackLayout>
-                    {filteredFilters
-                        .sort((a, b) => b.data.count - a.data.count)
-                        .slice(0, isShowingMore ? Infinity : maxShowing)
-                        .map((filter, i) => (
-                            <StackLayout
-                                className={styles.checkboxFilterItem}
-                                horizontal
-                                key={`${filterGroup.field}-${filter.id}-${filter.data.count}-${selectedFilters.length}-${i}`}
+                    {filteredFilters.slice(0, isShowingMore ? Infinity : maxShowing).map((filter, i) => (
```

Comment:
is it safe to remove the sort for all project?

Reply:
ES always sort the data from higher to lower right now. So it should be safe, I put a screenshot why I need to remove it.

We could maybe pass either a sort method down the row or activate it or not. But like I said right now ES does it for us

Reply:
👍